File: DuDoong-Domain/src/main/java/band/gosrock/domain/domains/order/exception/NotApprovalOrderException.java
package band.gosrock.domain.domains.order.exception;


import band.gosrock.common.exception.DuDoongCodeException;

public class NotApprovalOrderException extends DuDoongCodeException {

    public static final DuDoongCodeException EXCEPTION = new NotApprovalOrderException();

    private NotApprovalOrderException() {
        super(OrderErrorCode.ORDER_NOT_APPROVAL);
    }
}

File: DuDoong-Domain/src/main/kotlin/band/gosrock/domain/domains/order/exception/NotApprovalOrderException.kt
package band.gosrock.domain.domains.order.exception

import band.gosrock.common.exception.DuDoongCodeException

class NotApprovalOrderException private constructor() : DuDoongCodeException(OrderErrorCode.ORDER_NOT_APPROVAL) {
    companion object {
        @JvmField
        val EXCEPTION: DuDoongCodeException = NotApprovalOrderException()
    }
}
